test/controllers/rentals_controller_test.rb

```diff
@@ -0,0 +1,56 @@
+require "test_helper"
+
+describe RentalsController do
+  describe "zomg" do
+    it "returns a message" do
+      get zomg_path
+      body = JSON.parse(response.body)
+      body["Message"].must_equal "it works!"
+    end
+  end
+
+  describe "overdue" do
+    it "returns 200 if overdue rentals exit" do
+      get overdue_path
+      must_respond_with :ok
+    end
+
+    it "returns 404 if overdue rentals don't exit" do
```

Comment:
typo - exi*s*t 
Now that I think about it, 404 might not be the response you want here.  404 means that the information you were looking for was not found - there's some kind of problem.  What you want to be saying here is that the information/category was found, but that there are zero things in the category. Might make more sense to give the user some kind of message:  "There are no overdue rentals at this time".